charts/kube-ovn-v2/values.yaml

```
@@ -0,0 +1,719 @@
+  # @section -- Kube-OVN controller configuration
+  labels: {}
+  # -- Annotations to be added to kube-ovn-controller pods.
+  # @section -- Kube-OVN controller configuration
+  podAnnotations: {}
+  # -- Labels to be added to kube-ovn-controller pods.
+  # @section -- Kube-OVN controller configuration
+  podLabels: {}
+
+  # -- kube-ovn-controller resource limits & requests.
+  # ref: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+  # @section -- Kube-OVN controller configuration
+  resources:
+    requests:
+      cpu: "200m"
+      memory: "200Mi"
+    limits:
+      cpu: "1000m"
+      memory: "1Gi"
+
+  # -- Controller metrics configuration.
+  # @section -- Kube-OVN controller configuration
+  # @default -- "{}"
+  metrics:
+    # -- Configure the port on which the controller service will serve metrics.
+    # @section -- Kube-OVN controller configuration
+    port: 10660
+
+# -- Configuration for ovn-central, the daemon containing the northbound/southbound DBs and northd.
+# @section -- OVN-central daemon configuration
+# @default -- "{}"
+central:
+  # -- Annotations to be added to all top-level ovn-central objects (resources under templates/central)
+  # @section -- OVN-central daemon configuration
+  annotations: {}
+  # -- Labels to be added to all top-level ovn-central objects (resources under templates/central)
+  # @section -- OVN-central daemon configuration
+  labels: {}
+  # -- Annotations to be added to ovn-central pods.
+  # @section -- OVN-central daemon configuration
+  podAnnotations: {}
+  # -- Labels to be added to ovn-central pods.
+  # @section -- OVN-central daemon configuration
+  podLabels: {}
+
+  # -- ovn-central resource limits & requests.
+  # ref: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+  # @section -- OVN-central daemon configuration
+  resources:
+    requests:
+      cpu: "300m"
+      memory: "200Mi"
+    limits:
+      cpu: "3"
+      memory: "4Gi"
+
+  # -- ""
+  # @section -- OVN-central daemon configuration.
+  ovnNorthdProbeInterval: 5000
+  # -- ""
+  # @section -- OVN-central daemon configuration.
+  ovnNorthdNThreads: 1
+  # -- ""
+  # @section -- OVN-central daemon configuration.
+  ovnLeaderProbeInterval: 5
+
+# -- Configuration for kube-ovn-cni, the agent responsible for handling CNI requests from the CRI.
+# @section -- CNI agent configuration
+# @default -- "{}"
+agent:
+  # -- Annotations to be added to all top-level agent objects (resources under templates/agent)
+  # @section -- CNI agent configuration
+  annotations: {}
+  # -- Labels to be added to all top-level agent objects (resources under templates/agent)
+  # @section -- CNI agent configuration
+  labels: {}
+  # -- Annotations to be added to the agent pods (kube-ovn-cni)
+  # @section -- CNI agent configuration
+  podAnnotations: {}
+  # -- Labels to be added to the agent pods (kube-ovn-cni)
+  # @section -- CNI agent configuration
+  podLabels: {}
```

```suggestion
  template:
    annotations: {}
    labels: {}
```

I think this will make it more clear.

---

I copied that concept from Cilium's Helm Chart, I think the comment is pretty clear on what this value edits, but I can modify each daemon's value to reflect your change if you want to.

The idea is that each daemon has a bunch of config for it (service accounts, roles, pods...) and the "annotations/labels" value add annotations and labels to every resource of that daemon. And podLabels/podAnnotations only add to the actual pod of the daemon.